### ISSUE TITLE
Reimport doesn't update instanced scenes in the 3D editor viewport

### ISSUE THREAD

**Issue by mrjustaguy** [godotengine/godot#55420]
### Godot version

v4.0.dev.20211117.official [175690957]

### System information

Windows 11, Vulkan, Nvidia Quadro K2000 461.92

### Issue description

Been tweaking auto mesh LOD generation settings, and Reimporting, and I've noticed that after reimport, the Mesh wasn't changed in any way.. then I disabled Mesh LOD totally, reimported, and the mesh still had Auto LODs active

### Steps to reproduce

Import Mesh With Auto LOD Enabled
Disable Auto LOD in Import Settings
Reimport -> LODs still active and unchanged

Note: Restarting the Editor Loads the changes

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/14338.

**Comment by FlooferLand** [comment]
This issue has been happening to me as well regardless of LOD settings. Even happens in 3.x for me.
In my case I've had to close and reopen my scenes for the instanced meshes to update.
Godot 4 has a shortcut option to reload the current scene luckily, but it's still kinda irritating to do

In a more extreme case in 4.0 (beta 2), I've had to restart the entire engine after adding/changing around external materials for the materials to update.
